client/src/AddFriend.js

```diff
@@ -118,8 +120,7 @@ class AddFriend extends React.Component {
           {displayAlert(this.state)}
         </div>
 
-        <h5>Total Count</h5>
-        <h5>0</h5>
+        <CountDisplay location={this.props.location}/>
```

Comment:
What does location refer to?

Reply:
It's used for `location.pathname` to get the base URL for the API call, including the user secret. You can ctrl+f for it in CountDisplay